cobbler/remote.py

```diff
@@ -1845,6 +1846,7 @@ def modify_item(
         )
         obj = self.__get_object(object_id)
         self.check_access(token, f"modify_{what}", obj.name, attribute)
+        self.__check_items_ready(what)
```

Comment:
Is there a need to add this to more methods than `new` and `edit`?

Reply:
@tpw56j Since this PR is a bugfix I would love to finish the discussion here to make the related PRs smaller.

Reply:
@SchoolGuy I looked at this code again and am now not sure if it is necessary. The code is only needed to produce a user-friendly message instead of a timeout error. But in this case it may be false because not for every item type, complete deserialization of the collection leads to the inability to add/change/delete/rename objects. As a result, we may prevent the user from performing an operation that they could have performed without this check.
I also hope to add a background task that will deserialize collections when the cobbler starts, which will reduce the risk of getting a timeout error.
Do you think maybe we shouldn’t add this code here?

Reply:
@SchoolGuy 
I have removed this code.

Reply:
I would prefer if we would have some user-friendly message because of XML-RPCs inability to return the proper HTTP status code. This message then be checked for in an automation.

Reply:
@SchoolGuy This task turned out to be more difficult than I thought, maybe this is a topic for a separate PR?

Reply:
I am fine with that but then let's create an issue for tracking our efforts.